### DISCUSSION TITLE
Free To Use?

### DISCUSSION THREAD

**Discussion by DeProfCodes** [journeyapps/zxing-android-embedded#724] [Q&A] [answered]
Hello, I am developing an app that I need to have a custom scanner preview with buttons, something like 'continuous scan' on this project. My question is is this project free to use in my project and what is the best way to integrate this feature into my app? Add whole project into my app?

**Comment by rkistner** [comment, marked as answer]
Yes, this is free to use, licensed under the Apache License 2.0.

Instructions to use are in the readme - you should add the gradle dependency, don't add the source code of this project to your app.
